### ISSUE TITLE
Make test/demo page easily accessible online

### ISSUE THREAD

**Issue by mathiasbynens** [googlefonts/color-fonts#77]
https://github.com/googlefonts/color-fonts/blob/main/test.htm contains a test page. Have you considered making it available at `https://googlefonts.github.io/color-fonts/blob/main/test.htm`? (Could be done by enabling GitHub Pages.)

**Comment by M-a-c** [comment]
yeah if someone could also provide a download link too so people who don't want to build this eg designers, and others can just download it
